Sources/SQLKit/SQLRowDecoder.swift

```diff
@@ -1,8 +1,28 @@
-struct SQLRowDecoder {
-    func decode<T>(_ type: T.Type, from row: SQLRow, prefix: String? = nil) throws -> T
+public struct SQLRowDecoder {
+    public var keyPrefix: String? = nil
+    public var keyDecodingStrategy: KeyDecodingStrategy = .useDefaultKeys
+
+    func decode<T>(_ type: T.Type, from row: SQLRow) throws -> T
         where T: Decodable
     {
-        return try T.init(from: _Decoder(prefix: prefix, row: row))
+        return try T.init(from: _Decoder(row: row, options: options))
+    }
+
+    public enum KeyDecodingStrategy {
+        case useDefaultKeys
+        case convertFromSnakeCase
+        case custom(([CodingKey]) -> CodingKey)
+    }
+
+    fileprivate struct _Options {
+        let keyPrefix: String?
+        let keyDecodingStrategy: KeyDecodingStrategy
+    }
+
+    /// The options set on the top-level decoder.
+    fileprivate var options: _Options {
+        return _Options(keyPrefix: keyPrefix,
+                        keyDecodingStrategy: keyDecodingStrategy)
```

Comment:
Vapor style is all indents all params on new lines with 1 tab width. (You can see it elsewhere in this repo)